File: 7.js
const readline = require("readline");
const rl = readline.createInterface({
  input: process.stdin,
  output: process.stdout,
});
rl.question("Enter the age: ", function(input) {
  const age = parseInt(input);
  if (age >= 18) {
    console.log("eligible");
  } else {
    console.log("not eligible");
  }
  rl.close();
});

File: 7.cpp
#include <iostream>
using namespace std;

int main() 
{
    int age;
    cout << "Enter the age: ";
    cin >> age;
    if (age >= 18) 
    {
        cout << "eligible" << endl;
    }
    else 
    {
        cout << "not eligible" << endl;
    }
    return 0;
}